defender/gasless-nft-minting/README.md

```diff
@@ -0,0 +1,169 @@
+# Gasless NFT Minter
+
+This code will allow users to deploy an [ERC-2771-compatible](https://docs.openzeppelin.com/contracts/4.x/api/metatx#ERC2771Context) [ERC-1155 NFT](https://docs.openzeppelin.com/contracts/4.x/erc1155) contract and then use one Autotask to sign mint requests, and another Autotask to relay the request to a [trusted forwarder](https://docs.openzeppelin.com/contracts/4.x/api/metatx#MinimalForwarder) which will send the request to the NFT contract to mint an NFT for the user.
+
+## Setup
+
+This section will help you set up the following:
+- Acquire Defender and Relay keys
+- Add funds to the Relay
+- Deploy contracts
+  - `yarn deploy contract --contract-name MinimalForwarder`
+  - `yarn deploy contract --contract-name DemoNFT "0xMINIMAL_FORWARDER_ADDRESS"`
+- Update `config.dev.yml` with the 2 deployed contract addresses
+- Deploy the Autotasks with `serverless deploy`
+- Connect Relays to deployed Autotasks
+  - Contract Deployer Relay must be connected to the signing Autotask
+  - Any Relayer with funds can be connected to the Relay Autotask
+- Copy the 2 Autotask webhooks into the `config.dev.yml` file
+
+### Defender Account Setup
+
+- In your [Defender account](https://defender.openzeppelin.com/), select the Hamburger icon in the upper right corner and click on **Team API Keys**
+- In the Team API Keys screen, click **Create API Key**
+- Make sure that all Capabilities are selected
+- Click **Save**
+- Copy your API key and Secret key to a local file (you will **NOT** be able to view your Secret key again after this message box goes away)
+- Make sure that you really did copy your API key and Secret key to a local file
+- Check the box for **I’ve written down the Secret key** and select **Close**
+
+### Relay API Keys
+
+- In your [Defender account](https://defender.openzeppelin.com/), select the **◎ Relay** link on the upper left side to get to the Relay Dashboard
+- If you do not have a Relay on the network that you want to deploy to, create one:
+  - In the Relay Dashboard screen, click **Create Relayer** in the upper right corner
+  - Pick a meaningful name, choose the network that you want to deploy to, and click **Create**
+- Click on the existing Relay that you would like to use for deployments
+- Click on the settings gear/cog **⚙** at the top middle and choose **+ Create new API key**
+- Copy your Ethereum Address, API key, and Secret key to a local file (you will **NOT** be able to view your Secret key again after this message box goes away)
+- Make sure that you really did copy your API key and Secret key to a local file
+- Check the box for **I’ve written down the secret key** and select **Close**
+
+### Add Funds to the Relay
+
+- In your [Defender account](https://defender.openzeppelin.com/)
+- Click on the **Relay** link on the left side
+- Find the Relay that deployed the NFT contract, click on the **copy address** button
+- Transfer funds to that address to cover the deployment and NFT minting costs
```

Comment:
I think this could be a bit misleading. At this point, it's saying to find the Relay that has already deployed the NFT contract, but then saying to transfer funds to the Relay address to cover the cost of deployment.

You could replace line 46 with something like `Find the Relay that will be used to deploy the NFT contract`

Reply:
Thanks, it has been updated.